test/ESP3Parser.test.js

```diff
@@ -65,6 +67,7 @@ describe('serialport enocean parser', function () {
 
         esp3parser.write(largeAndMessyByteStream)
         assert.equal(spy.callCount, telegrams.length, 'Received unexpected count of packets.')
+        assert.equal(error_spy.callCount, telegrams.length, 'Received unexpected count of packets.')
       })
 
```

Comment:
Move to separate test case
```suggestion

      it('error must be emitted on failed CRC8 checks for ESP3 Packet header', function () {
        const spy = sinon.spy()
        esp3parser.on('error', spy)
        esp3parser.write(Buffer.from('55af600ffd9b7812001f', 'hex'))
        assert.isTrue(spy.calledOnce, 'Error is not emitted on fails in ESP3 Packet header.')
        assert.equal(spy.getCall(0).args[0].code, 1, 'Wrong error code on failing CRC8 check for header.')
      })

      it('error must be emitted on failed CRC8 Checks for ESP3 Packet datas', function () {
        const spy = sinon.spy()
        esp3parser.on('error', spy)
        esp3parser.write(Buffer.from('55000a0701eba540300287ffd9b7e50001ffffffff440000', 'hex'))
        assert.isTrue(spy.calledOnce, 'Error is not emitted on fails in ESP3 Packet header.')
        assert.equal(spy.getCall(0).args[0].code, 2, 'Wrong error code on failing CRC8 check for datas.')
      })
```